README.md

```diff
@@ -1,98 +1,518 @@
+Of particular note, are `audit.mixin` and `actions.mixin`. 
+These add the actions functionality (from `./lib/actions.coffee`)
+and audit functionality (from pantheon-helpers) to your design doc.
+We will be modifying `./lib/actions.coffee` later to 
+create the actions our app can perform.
+
+If you look in `$SISYPHUS/lib/design_docs/boulder`,
+you will see some files that are not in source. First, is the `kanso.json` file, this is similar to node.js `package.json` or a bower `bower.json` file.
+It tells kanso what to package up and send to couchdb.
+
+Next, in the `lib` subdirectory you will see a copy of underscore, and a symlink to the `pantheon-helpers/lib/design_docs` folder. Any files that you want to reference in your design doc must be in the `boulder` directory, otherwise Kanso can't package them up.
+So we add them here.
+
+Now that we have created our design document,
+we have associate it with a type of database.
+To do this, we create a new file at
+$SISYPHUS/src/design_docs/boulders.coffee with the following contents:
+
+    module.exports = ['boulder']
+
+This tells Pantheon to add the `boulder` design doc to every
+single database in CouchDB that starts with the string `boulders`.
+If we wanted all databases that start with `boulders` to also
+have another design doc installed, we would add the name of the desired design doc to the experted array.
+
+### 3. Design the Sisyphus microservice
+Rolling the rock up the hill takes a long time (in web time): 2 minutes. When we make a request to roll the rock up the hill, we do not want to have to wait two minutes for a response. Instead, we would like to receive a response instantly that our request to roll the rock up the hill has been accepted and is being processed. Then we want a background process to actually roll the rock up the hill for two minutes.
+
+Pantheon-helpers makes it easy to build this sort of decoupled application. First, let's figure out what our data is going to look like, then let's figure out what actions we want to be able to perform on that data. Finally, we'll implement everything.
+
+Our rock is going to be represented by a json document.
+We want to know whether it's rolling up the hill,
+rolling down the hill, or at the bottom of the hill.
+We're also curious about Zeus's reaction to events as they unfold.
+Thus, our json document will look like this:
+
+    { "_id": "document ID"
+    , "type": "boulder"
+    , "status": "rolling up|rolling down|at bottom"
+    , "zeus": 
+      { "is": "expectant|delighted|satisfied|mirthful|vengeful"
+      }
+    }
+
+That's pretty easy! 
+We don't even have to worry about the _id.
+CouchDB will create one for us if we don't set it explicitly.
+
+We want to be able to perform four different actions on this data:
+  1. create a new boulder ('b+')
+  2. start rolling the rock up the hill ('bu')
+  3. make the rock slip away and roll back down the hill ('bd')
+  4. bring the boulder to rest at the bottom of the hill ('br')
```

Comment:
Should the four action names above be wrapped with backticks instead of single quotes?